packages/server/beaver_triple_service/triple_generator/triple_generator_test.go

```diff
@@ -42,16 +46,12 @@ func getTriplesForParallel(t *testing.T, partyId uint32, amount uint32, jobNum u
 	t.Helper()
 	for jobId := uint32(0); jobId < jobNum; jobId++ {
 		t.Run(fmt.Sprintf("TestTripleGenerator_Job%d", jobId), func(t *testing.T) {
-			triples, err := tg.GetTriples(claims, jobId, partyId, amount, triple_type)
+			triples, err := tg.GetTriples(claims, jobId, partyId, amount, triple_type, -1)
```

Comment:
Since only the `requestId=-1` case is tested, I would like to see the following new test added.
```Go
func Test~~~(t *testing.T){
  // Same requestID test
  t1, err := tg.GetTriples(claims, jobId, partyId, amount, triple_type, 1)
  if err != nil {...}
  t2, err := tg.GetTriples(claims, jobId, partyId, amount, triple_type, 1)
  if err != nil {...}
  if t1 != t2 {....} // same check
}
func Test~~~(t *testing.T){
  // Different requestID test
  t1, err := tg.GetTriples(claims, jobId, partyId, amount, triple_type, 1)
  if err != nil {...}
  t2, err := tg.GetTriples(claims, jobId, partyId, amount, triple_type, 2)
  if err != nil {...}
  if t1 == t2 {....} // diff check
}
```

Reply:
add such tests (`TestSameRequestId` and `TestDifferentRequestId`) and `TestOutRangePartyId`